shard.yml

```diff
@@ -1,7 +1,8 @@
 name: ameba
-version: 0.13.2
+version: 0.13.3
+
+crystal: 1.0
```

Comment:
Is there any breaking change in Ameba which doesn't support crystal 0.29.0 now? I think here the idea is to specify the latest compatible crystal version (which is 0.29.0 in my understanding)

Reply:
IIRC minimum supported version is `0.34` or `0.35`